packages/flutter/test/foundation/leak_tracking.dart

```diff
@@ -122,6 +122,7 @@ Future<void> _withFlutterLeakTracking(
         asyncCodeRunner: asyncCodeRunner,
         leakDiagnosticConfig: config.leakDiagnosticConfig,
         shouldThrowOnLeaks: false,
+        gcCountBuffer: 1,
```

Comment:
Are there any downsides to this?

Reply:
What does this property actually control? It's documentation wasn't all that helpful: https://github.com/dart-lang/leak_tracker/blob/6e3f57c8ce7609882ea2209b3d07937b73823e23/pkgs/leak_tracker/lib/src/leak_tracking/orchestration.dart#L48

Reply:
Improvement for documentation: https://github.com/dart-lang/leak_tracker/pull/100

Reply:
Downside can be false positive. Already got some failures, so increased to 2.